src/service/payment.service.ts

```diff
@@ -98,7 +104,15 @@ const setPaymentRequest = (transactionId: string, overseasEntityId: string, base
 
   // Once payment has been taken, the platform redirects the user back to the application,
   // using the application supplied `redirectUri`.
-  const redirectUri = `${baseURL}${TRANSACTION}/${transactionId}/${OVERSEAS_ENTITY}/${overseasEntityId}/${PAYMENT}`;
+  let redirectUri = `${baseURL}${TRANSACTION}/${transactionId}/${OVERSEAS_ENTITY}/${overseasEntityId}/${PAYMENT}`;
+
+  // TODO Remove this and the check for being on the registration journey when ids are in the Update journey URLs
+  const isRegistration: boolean = baseURL.includes(LANDING_URL);
+
+  if (isActiveFeature(FEATURE_FLAG_ENABLE_REDIS_REMOVAL) && isRegistration) {
```

Comment:
Now wondering if this needed to be done @markpit , as the existing redirect URL had the transaction id and OE id in it...? I suppose it's more consistent doing this.

Reply:
old url had overseas-entity instead of submission so would have been different to all the other urls